apps/website/src/pages/index.tsx

```diff
@@ -239,11 +239,11 @@ const Home: NextPage = () => {
               </p>
               <p className="my-4 text-lg">
                 Alveus is a nonprofit organization founded by Maya Higa that
-                functions as an exotic animal sanctuary and as a virtual
-                education center facility to provide permanent homes to
-                non-releasable exotic animals. These animals function as
-                ambassadors, so viewers can watch their journeys, get to know
-                the animals, and gain an appreciation for their species.
+                functions as a wildlife sanctuary and as a virtual education
+                center facility to provide permanent homes to non-releasable
+                animal ambassadors. These animals function as ambassadors, so
+                viewers can watch their journeys, get to know the animals, and
+                gain an appreciation for their species.
```

Comment:
The sentences here feel a bit duplicative now, open to suggestions (or can leave it as-is, not a big deal)

Reply:
smth like Viewers are able to watch ambassador's journeys, get to know them and gain appreciation for their species?

Reply:
I'm going to leave this as-is for now so we can get this in, but feel free to open a PR if you do want to take a stab at rewording :)